### PR TITLE
More changes needed for aarch64

### LINKED ISSUE #5306: Add support for Apple M1 ?
Appreciate your work, thank you.

I am unable to install on Apple M1 using Eclipse 4.20M3 for Arm64/AArch64 :

```
Cannot complete the install because some dependencies are not satisfiable
  org.eclipse.equinox.p2.iu; net.resheim.eclipse.launcher.feature.group [3.0.2.202004261020,3.0.2.202004261020] cannot be installed in this environment because its filter is not applicable.
```

Maybe the x86_64 feature filter needs to be removed ? Just guessing.

Thanks again.

### PR DESCRIPTION
**Pull request by guw**
This closes #5306

### CODE REVIEW

**Comment by guw** [comment]
@turesheim With these changes I was able to build it locally.

File: default.tpd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 target "Eclipse Launcher Target Platform" with source, requirements
 
 // Base dependencies
-location "http://download.eclipse.org/releases/oxygen/" {
+location "http://download.eclipse.org/releases/2021-12/" {
```
Comment (guw) [inline default.tpd:4]: Oxygen does not have `aarch64` support. Need a more recent version of Eclipse.

File: pom.xml [hunk 2]
```diff
@@ -84,6 +84,11 @@
 							<ws>cocoa</ws>
 							<arch>x86_64</arch>
 						</environment>
+						<environment>
+							<os>macosx</os>
+							<ws>cocoa</ws>
+							<arch>aarch64</arch>
+						</environment>
```
Comment (guw) [inline pom.xml:91]: Target platform needs to be configured for `aarch64`

File: net.resheim.eclipse.launcher.macosx/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -13,4 +13,4 @@ Bundle-Vendor: %Bundle-Vendor
 Bundle-Localization: OSGI-INF/l10n/bundle
 Export-Package: net.resheim.eclipse.launcher.macosx,
  net.resheim.eclipse.launcher.macosx.handlers;x-friends:="net.resheim.eclipse.launcher.tests"
-Eclipse-PlatformFilter: (& (osgi.ws=cocoa) (osgi.os=macosx) (osgi.arch=x86_64))
+Eclipse-PlatformFilter: (& (osgi.ws=cocoa) (osgi.os=macosx))
```
Comment (guw) [inline net.resheim.eclipse.launcher.macosx/META-INF/MANIFEST.MF:16]: A missed filter condition. `osgi.arch` not needed to filter here. Code only needs SWT.

**Comment by turesheim** [comment]
Awesome, thanks @guw !

**Pull request merged**